packages/core/src/checkout/checkout-service.spec.ts

```diff
@@ -1488,11 +1488,29 @@ describe('CheckoutService', () => {
             );
 
             const container = 'checkout.extension';
-            const region = ExtensionRegion.ShippingShippingAddressFormBefore;
+            const region = 'shipping.shippingAddressForm.before';
 
             await checkoutService.renderExtension(container, region);
 
             expect(extensionActionCreator.renderExtension).toHaveBeenCalledWith(container, region);
         });
     });
+
+    describe('#handleExtensionCommand()', () => {
+        it('handles extension command', async () => {
+            const checkoutServiceMock: any = checkoutService;
+            const listenMock = jest.fn();
+
+            checkoutServiceMock._extensionMessenger = {
+                listen: listenMock,
+            };
```

Comment:
A workaround to test `extensionMessenger` as a private property. 😹

Reply:
Why don't you just inject an instance of `ExtensionMessenger` when constructing `checkoutService` in the `beforeEach`?

Reply:
I was trying to avoid modifying the constructor to inject an instance of extensionMessenger as the constructor mostly receives actionCreators instead. Now I have injected extensionMessenger, this workaround is no longer needed.